### DISCUSSION TITLE
How can I get current hunger info?

### DISCUSSION THREAD

**Discussion by GK64** [PrismarineJS/mineflayer#1925] [Q&A] [answered]
I'm creating an automatic eating, but I'd like to see the current hunger information when I type 'Hunger info' in the public chat.
I found 'bot.food()' but I don't understand how to use it.

Is there any other api to get information easily? ex. health, effects, hunger.

```                                                               
bot.on('chat', (username,message) => {

    if (message = "hunger info"){
    bot.chat('Hunger = ')
  }
  return
  })
})
```

**Comment by aesthetic0001** [comment, marked as answer]
bot.food should do it. Please ask questions like this in discussions

**Comment by amoraschi** [comment]
It's a property not a function

**Comment by aesthetic0001** [comment]
Yeah
